src/Bridge/Symfony/Bundle/Resources/config/services.xml

```diff
@@ -14,5 +14,13 @@
         <service id="Elao\Enum\Bridge\Symfony\Serializer\Normalizer\EnumNormalizer">
             <tag name="serializer.normalizer" priority="-800" />
         </service>
+
+        <service id="Elao\Enum\Bridge\Symfony\Translation\Extractor\EnumExtractor">
+            <argument>%elao_enum.translation_extractor.paths%</argument>
```

Comment:
In order to avoid exposing parameters just for this purpose, we're used to define empty arguments here:

```xml
<argument /> <!-- argument name/short descr -->`
```

 and replace the definition arguments (with `replaceArgument`) directly from the bundle extension. 
Could you please update this accordingly? 🙏

Reply:
Done, thanks for the feedback.